<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgInconformity</class>
 <widget class="QDialog" name="CDlgInconformity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车卡不符</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(D:/WinParkUI/debug/Image/NewIcon/Inconformity.JPG);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>56</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
   <property name="title">
    <string>车牌识别用户信息</string>
   </property>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(none);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>按该用户放行</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(none);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>卡号：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo10">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>卡类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo11">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo12">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>电话：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo13">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>车牌：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo14">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>56</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
   <property name="title">
    <string>当前刷卡用户信息</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(none);</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>卡号：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo00">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>卡类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo01">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo02">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>电话：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo03">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
         <property name="text">
          <string>车牌：  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtInfo04">
         <property name="styleSheet">
          <string notr="true">background-image: url(none);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(none);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>按该用户放行</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string>%1口：车牌与卡不符</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>出口：车牌与卡不符</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(none);</string>
   </property>
   <property name="text">
    <string>不放行</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
